xsuite/src/cli/installRustCmd.ts

```diff
@@ -12,12 +12,17 @@ export const addInstallRustCmd = (cmd: Command) => {
     .description(
       `Install Rust with rustup: toolchain ${rustToolchain} & target ${rustTarget}.`,
     )
+    .option(
```

Comment:
I think we could set a default value here? And this way `toolchain` would then be always defined

Reply:
Done

Reply:
We should rather use:

https://github.com/tj/commander.js?tab=readme-ov-file#default-option-value

And line 23 we should not need to provide the default value